core/startup/config_file_watcher.go

```diff
@@ -15,18 +15,31 @@ import (
 type WatchConfigDirectoryCloser func() error
 
 func ReadApiKeysJson(configDir string, appConfig *config.ApplicationConfig) error {
-	fileContent, err := os.ReadFile(path.Join(configDir, "api_keys.json"))
-	if err == nil {
-		// Parse JSON content from the file
-		var fileKeys []string
-		err := json.Unmarshal(fileContent, &fileKeys)
-		if err == nil {
-			appConfig.ApiKeys = append(appConfig.ApiKeys, fileKeys...)
-			return nil
-		}
-		return err
-	}
-	return err
+    fmt.Println("Config directory:", configDir) // Print the configDir

```

Comment:
don't use fmt.Println, use something like this instead:

`log.Debug().Str("config_dir", configDir).Msg("Configuration directory path")`